src/api/app/views/webui/users/notifications/_update.js.erb

```diff
@@ -1,3 +1,4 @@
 $('#filters').html("<%= j(render partial: 'notifications_filter', locals: { filter: notifications_filter }) %>");
 $('#notifications-list').html("<%= j(render partial: 'notifications_list', locals: { notifications: notifications, selected_filter: notifications_filter.selected_filter }) %>");
 $('#flash').html("<%= escape_javascript(render(layout: false, partial: 'layouts/webui/flash', object: flash)) %>");
+$('#notifications-counter').html("<%= j(render partial: 'layouts/webui/responsive_ux/unread_notifications') %>");
```

Comment:
While `j` and `escape_javascript` are the same, `escape_javascript` is much more readable, so let's use it :slightly_smiling_face:

```suggestion
$('#notifications-counter').html("<%= escape_javascript(render partial: 'layouts/webui/responsive_ux/unread_notifications') %>");
```